deploy/009-deploy-vtokens.ts

```diff
@@ -21,6 +21,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { tokensConfig, poolConfig, preconfiguredAddresses } = await getConfig(hre.network.name);
 
   const { isTimeBased, blocksPerYear } = getBlockOrTimestampBasedDeploymentInfo(hre.network.name);
+  const maxBorrowRateMantissa = BigNumber.from(0.0005e16);
```

Comment:
We could move this config to `deploymentConfig.ts` and make it per network

Reply:
What would be the corresponding values for other networks.

Reply:
* Ethereum: I would set the same value (because we have already deployed the VTokens to Ethereum using this constant)
* Sepolia: same as Ethereum
* opBNB mainnet and testnet: same as Ethereum, to not generate a dependency with the deployment of the markets to that networks
* Arbitrum: I wouldn't define it now, but it could be `0.00016e16` (`0.0005e16 / 3`, because in BNB there is one block every 3 seconds)

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/324/commits/c749e0d6d301357876806274d244e06da0cb8108